src/Mafia/Home.hs

```diff
@@ -62,6 +62,7 @@ installBinary package deps = do
   -- We can remove this at some point
   -- NOTE: This makes no attempt at being threadsafe
   whenM (liftIO (doesFileExist prefix)) (removeFile prefix)
+  createDirectoryIfMissing True prefix
```

Comment:
Would this throw if there was a race and the directory already existed? I wouldn't normally worry about that for something like this, but we are going to the trouble of taking a lock.